sunpy/coordinates/frames.py

```diff
@@ -87,6 +87,10 @@ def _frame_parameters():
                        "        it must be a solar system body that can be parsed by\n"
                        "        `~sunpy.coordinates.ephemeris.get_body_heliographic_stonyhurst`\n"
                        "        at the time ``obstime``. Defaults to Earth center.")
+    ret['rsun'] = ("rsun : `~astropy.units.Quantity`\n"
+                   "        The physical (length) radius of the Sun. Used to convert a 2D\n"
```

Comment:
```suggestion
                   "        The radius of the Sun. Used to convert a 2D\n"
```
? Unless having "physical (length)" here has a technical meaning I've not come across?

Reply:
The wording, which I did not modify from the existing wording, is presumably to distinguish between the "physical (length)" radius and the "apparent (angle)" radius.

Reply:
I'll change it to "The radius of the Sun in length units."